apps/src/util/userSectionClient.js

```diff
@@ -1,4 +1,5 @@
 /* globals $ */
+import $ from 'jquery';
```

Comment:
Rather than importing jquery here, you can stub userSectionClient in the test file. 

Example:
```
describe('NetSimLobby', () => {
  let rootDiv, netsim, getUserSectionsStub;
  beforeEach(() => {
    getUserSectionsStub = sinon.stub(userSectionClient, 'getUserSections');
    ...
  });

  afterEach(() => {
    userSectionClient.getUserSections.restore();
  });
```
You'll still be able to use the `expect` you have in line 26.